packages/js/customer-effort-score/src/components/customer-effort-score-tracks/index.js

```diff
@@ -69,37 +67,21 @@ function _CustomerEffortScoreTracks( {
 		return null;
 	}
 
-	// We only want to return null early if the modal was already shown
-	// for this action *before* this component was initially instantiated.
-	//
-	// We want to make sure we still render CustomerEffortScore below
-	// (we don't want to return null early), if the modal was shown for this
-	// instantiation, so that the component doesn't go away while we are
-	// still showing it.
-	if (
-		cesShownForActions &&
-		cesShownForActions.indexOf( action ) !== -1 &&
-		! modalShown
-	) {
-		return null;
-	}
```

Comment:
I do think we want to keep this condition here, I am not a 100% sure about the `! modalShown` logic, that could potentially be removed, I don't quite understand that part of the logic. So maybe its worth to also keep that.

Reason for needing this is that the ces could still be triggered from the `client` in which case we did have to check `cesShownForActions` first. One way to test this is the `product_add_view` action, by doing:
1.  Go to **Products > Add new**
2. Add some changes without saving
3. Click **WooCommerce > Home**
4. Click **Leave** when it prompts you about the unsaved changes
5. Notice the `notice` that shows up, currently this notice shows up every time, adding the above logic will cause it to only show once.

BTW I found this by git blaming the above code and looking at the PR that added it, I guess I had added/updated it as part of this PR: https://github.com/woocommerce/woocommerce/pull/35728 ( which includes the above scenario ).